source/languages/en/riak/dev/advanced/bucket-types.md

```diff
@@ -37,9 +37,9 @@ with a few crucial differences:
   those configurations to as many buckets as you wish, whereas the
   previous system required configuration to be set on a per-bucket basis
 * Nearly all bucket properties can be updated using bucket types, with
-  two exceptions: the `datatype` and `consistent` properties related to
-  [[Riak Data Types|Data Types]] and [[strong consistency]],
-  respectively
+  three exceptions: the `datatype`, `consistent`, and `write_once`
+  properties related to [[Riak Data Types|Data Types]],
+  [[strong consistency]], and [[Write Once Buckets]] respectively
```

Comment:
I would edit this sentence to read: 
"Nearly all bucket properties can be updated using bucket types, except the `datatype`, `consistent`, and `write once` properties related to..."